src/job/summary.ts

```diff
@@ -0,0 +1,12 @@
+import { CronJob } from "cron";
+import { init } from "../adapters/summarize/summarize";
+
+export const summaryJob = new CronJob(
+  "1 2 * * sun",
+  () => {
+    init();
+  },
+  null,
+  false,
+  "Atlantic/Reykjavik" //GMT-UTC matching timezone
+);
```

Comment:
I have a couple of issues with this implementation. Unfortunately you were not apart of my conversations with @0xcodercrane and some others regarding CRON but basically the conclusion is that we will not be using or relying on CRON. Instead, for timed events like these we have a "wildcard post event handler" which runs after every time this bot is invoked. Remember, the bot is only invoked as a response to an "event" e.g. a new comment, a new label, a new pull request etc. 

The idea is to always check after every event if it should post a summary. If not, presumably it should add to the summary queue. In this same logic it should also check if it posted the summary of the weeks that have passed (we can max it to just the last week, so that it won't post infinite summaries since the beginning of time.)

Also I'm skeptical that this implementation will work at all because the bot must be actively performing a task at exactly the same time as `""1 2 * * sun"` to catch this. The Action will only run for the duration of the function which will likely be less than a minute at a time.

Reply:
So I have replaced the CronJob with wildcard selector @pavlovcik